manifests/r/RARLab/WinRAR/6.20.0/RARLab.WinRAR.installer.yaml

```
@@ -0,0 +1,95 @@
+PackageVersion: 6.20.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: install
+Commands:
+- winrar
+FileExtensions:
+- 7z
+- arj
+- bz2
+- cab
+- gz
+- iso
+- jar
+- lz
+- lzh
+- rar
+- tar
+- uue
+- xz
+- z
+- zip
+- zipx
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x64-620.exe
+  InstallerSha256: 859B2CD35B47D91DA08ED0547AD6348E1543F153D8E225660B62A9AF6C986044
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x32-620.exe
+  InstallerSha256: 0A2E517907034D55C971D9391D0AACE868D73B5B09F9879BB1A87948D9DA7BAF
+- InstallerLocale: fr-FR
+  Architecture: x64
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x64-620fr.exe
+  InstallerSha256: ED9FB1356F419FD6F7A6017523BE2D3B23A8B335FC480F157849822166A5B83B
+- InstallerLocale: fr-FR
+  Architecture: x86
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x32-620fr.exe
+  InstallerSha256: 04DAAEB7EE916D0919F3E6E52FBC55529A677BD0F5DA5CF27FC48635E1EBB713
+- InstallerLocale: de-DE
+  Architecture: x64
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x64-620d.exe
+  InstallerSha256: B38D3DA2AE966FA5D5B860BFC55400BCEE7A16FE051D7DF969FC543274326E97
+- InstallerLocale: de-DE
+  Architecture: x86
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x32-620d.exe
+  InstallerSha256: C16E1B37CC60A908F7237D695CFE63F0A4FE13347F1D5DBC9071CAF0D3238157
+- InstallerLocale: it-IT
+  Architecture: x64
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x64-620it.exe
+  InstallerSha256: 7E6E970D5CE8188E07B9B5AB0CA38088296261E76A1E6697FFF2ED33F7EA72B0
+- InstallerLocale: it-IT
+  Architecture: x86
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x32-620it.exe
+  InstallerSha256: 9C04E053ECAE85771783795FB7781501E8864C40C7E788A006C8855921D69A43
+- InstallerLocale: pt-PT
+  Architecture: x64
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x64-620pt.exe
+  InstallerSha256: 10431E2530D59C71C1A105A215D9620DC8A62C33732CC1D076F2806BCD76C824
+- InstallerLocale: pt-PT
+  Architecture: x86
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x32-620pt.exe
+  InstallerSha256: 5E0E0F9F7BD9796329DD2A8F8CC6BCFFECCAE5E6D91986EAED4DC048FB73589C
+- InstallerLocale: es-ES
+  Architecture: x64
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x64-620es.exe
+  InstallerSha256: 28E593ED3BD2EA090EFFFFCF24B65AA56965BB92FDF73C25CB3C96C632F7CD2E
+- InstallerLocale: es-ES
+  Architecture: x86
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x32-620es.exe
+  InstallerSha256: 109CEA214AC974C8AD86CCB90117701AF177FC62B3ED51F20676278644767B13
+- InstallerLocale: sv-SE
+  Architecture: x64
+  InstallerUrl: https://www.rarlab.com/rar/winrar-x64-620sw.exe
+  InstallerSha256: CDE7A5924094D49C7E6E1E30239C03771753C878E324D89593B5BEC138D0654F
+- InstallerLocale: sv-SV

```

```suggestion
- InstallerLocale: sv-SE
```
Looks like you forgot to fix the locale for the x64 version